### DISCUSSION TITLE
Discrepancy in UI language

### DISCUSSION THREAD

**Discussion by bheeshmpita** [FreeTubeApp/FreeTube#4541] [Q&A] [answered]
I have set my UI language to English (Settings->General settings->Language preference) but the video upload details like the time when was it was uploaded (mentioned just under the video title) is some foreign language. eg: _vor 10 Stunden_ which when translated, was found to be German. Neither i am in Germany nor I'm using any VPN.
How can i fix this? 
Thanks

**Comment by bheeshmpita** [comment]
@Svallinn @efb4f5ff-1298-471a-8973-3d47447115dc any suggestions please.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We do not change your languages settings based on IP so i dont understand how you are seeing this. 

1. Are you seeing this on all videos or just a certain one? 
2. You mentioned u have set your language to English, do you mean US or UK?
3. Please provide screenshots

**Comment by bheeshmpita** [comment]
Thanks @efb4f5ff-1298-471a-8973-3d47447115dc for replying
1. Yes, my homepage is subscriptions page
2. Eng UK
3. 
![ksnip_26012024-160110](https://github.com/FreeTubeApp/FreeTube/assets/90836004/0882b63a-7176-43a1-b165-9b2002ecbd84)

On further investigation, i found that my current invidious instance set to the default one is from Germany (https://invidious.nerdvpn.de). So that could be the reason.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hmm very weird, i tried to recreate this using your settings but it is still being shown normally on my end.

**Comment by absidue** [comment, marked as answer]
This bug was fixed in #4752 :)
